### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by sam1294**
Initial Commit to add code for k8res metrics implementation.

### CODE REVIEW

File: resconfig/res.yaml
Comment (Mona3087) [inline resconfig/res.yaml:1]: Let's cleanup this file to remove any previous custom resources and come up with some examples relevant for K8s

**Review by Mona3087 — COMMENTED** [review]
res.yaml needs change and do we need to use different port than 9805 for open sourcing it

File: resconfig/res.yaml
Comment (sam12941) [reply resconfig/res.yaml:1]: Done.

Comment (sam12941) [reply resconfig/res.yaml:1]:
```
# HELP etcd_status THis is an example of fetching state of Etcd Cluster.
# TYPE etcd_status gauge
etcd_status{name="quark-etcd-cluster",phase="Running"} 3
```

**Comment by sam12941** [comment]
I think 9805 should be fine.

**Pull request merged**